build/pause/Makefile

```diff
@@ -17,7 +17,7 @@
 REGISTRY ?= staging-k8s.gcr.io
 IMAGE = $(REGISTRY)/pause
 
-TAG ?= 3.10
+TAG ?= 3.10.1
```

Comment:
confirming we've actually had a patch revision before:
```console
$ crane ls registry.k8s.io/pause
0.8.0
1.0
2.0
3.0
3.1
3.10
3.2
3.3
3.4.1
3.5
3.6
3.7
3.8
3.9
go
latest
sha256-7031c1b283388d2c2e09b57badb803c05ebed362dc88d84b480cc47f72a21097.sig
sha256-9001185023633d17a2f98ff69b6ff2615b8ea02a825adffa40422f51dfdcde9d.sig
sha256-ee6521f290b2168b6e0935a181d4cff9be1ac3f505666ef0e3c98fae8199917a.sig
test
test2
```

Reply:
Initially I went with 3.11 but saw the 3.4.1
I'm happy to keep this at 3.11 or 3.10.1 - whichever makes more sense.

Reply:
I just couldn't remember a x.y.1 patch version, which makes sense given it has only happened once, I figured others might also wonder about this.